v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/options/ProtegrityDataTokenizationOptions.java

```diff
@@ -66,6 +66,11 @@
 
     void setBigTableKeyColumnName(String bigTableKeyColumnName);
 
+    @Description("Column family name to use in Cloud BigTable.")
+    String getBigTableColumnFamilyName();
+
+    void setBigTableColumnFamilyName(String bigTableColumnFamilyName);
+
```

Comment:
Shouldn't we use each possible column from the data as a separate column family? I.e. `{name: some_name, surname: some_surname}` comes up with 2 column families - name and surname.

Reply:
Column families should be present in the table, when the job starts
We cannot create them dynamically

Reply:
Got it